classes/Task/Update/UpdateFiles.php

```diff
@@ -85,7 +85,7 @@ public function run(): int
                 break;
             }
         }
-        $this->container->getState()->setProgressPercentage(
+        $this->container->getUpdateState()->setProgressPercentage(
```

Comment:
on more time baby

Reply:
Done in `warmUp` method but not in `run() as there is only one occurrence of the state in it.